feature/home/src/main/java/com/epmedu/animeal/home/presentation/viewmodel/HomeViewModel.kt

```diff
@@ -40,6 +40,8 @@ class HomeViewModel @Inject constructor(
         is HomeScreenEvent.FeedingPointSelected -> selectFeedingPoint(event)
         is HomeScreenEvent.FeedingPointFavouriteChange -> changeFavouriteFeedingPoint(event)
         is HomeScreenEvent.UserCurrentGeolocationRequest -> changeGpsSetting()
+        is HomeScreenEvent.ShowWillFeedDialog -> updateWillFeedDialog(true)
```

Comment:
WillFeedDialog.show() or showWillFeedDialog separate methods

Reply:
Ok, I did it cause eventually detekt will complain about max number of methods(14)